### PR TITLE
feat: Carousel support arrows for switching

### PR DESCRIPTION
**Pull request by afc163**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

close https://github.com/ant-design/ant-design/issues/12479

### 💡 Background and solution

Carousel 支持 arrows 功能。

图片

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Carousel support `arrows` to switching between slides.          |
| 🇨🇳 Chinese |  Carousel 现在支持 `arrows` 来开启箭头切换功能。        |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/carousel/index.zh-CN.md [hunk 1]
```diff
@@ -31,6 +32,7 @@ demo:
 
 | 参数 | 说明 | 类型 | 默认值 | 版本 |
 | --- | --- | --- | --- | --- |
+| arrows | 是否显示箭头 | boolean | false |  |
```
Comment (afc163) [inline components/carousel/index.zh-CN.md:35]:
```suggestion
| arrows | 是否显示箭头 | boolean | false | 5.17.0 |
```

File: components/carousel/index.en-US.md [hunk 2]
```diff
@@ -30,6 +31,7 @@ Common props ref：[Common props](/docs/react/common-props)
 
 | Property | Description | Type | Default | Version |
 | --- | --- | --- | --- | --- |
+| arrows | Whether to show switch arrows | boolean | false |  |
```
Comment (afc163) [inline components/carousel/index.en-US.md:34]:
```suggestion
| arrows | Whether to show switch arrows | boolean | false | 5.17.0 |
```

File: components/carousel/index.zh-CN.md [hunk 3]
```diff
@@ -23,6 +23,7 @@ demo:
 <code src="./demo/position.tsx">位置</code>
 <code src="./demo/autoplay.tsx">自动切换</code>
 <code src="./demo/fade.tsx">渐显</code>
+<code src="./demo/arrows.tsx">切换箭头</code>
```
Comment (afc163) [inline components/carousel/index.zh-CN.md:26]:
```suggestion
<code src="./demo/arrows.tsx" version="5.17.0">切换箭头</code>
```

File: components/carousel/index.en-US.md [hunk 4]
```diff
@@ -22,6 +22,7 @@ demo:
 <code src="./demo/position.tsx">Position</code>
 <code src="./demo/autoplay.tsx">Scroll automatically</code>
 <code src="./demo/fade.tsx">Fade in</code>
+<code src="./demo/arrows.tsx">Arrows for switching</code>
```
Comment (afc163) [inline components/carousel/index.en-US.md:25]:
```suggestion
<code src="./demo/arrows.tsx" version="5.17.0">Arrows for switching</code>
```

File: components/carousel/demo/arrows.tsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+import { Carousel } from 'antd';
+
+const contentStyle: React.CSSProperties = {
+  margin: 0,
+  height: '160px',
+  color: '#fff',
+  lineHeight: '160px',
+  textAlign: 'center',
+  background: '#364d79',
+};
```
Comment (Wxh16144) [inline components/carousel/demo/arrows.tsx:11]: 颜色是不是应该用 token

Comment (afc163) [reply components/carousel/demo/arrows.tsx:11]: 演示而已，应该没必要。

File: components/carousel/style/index.ts [hunk 6]
```diff
@@ -15,26 +15,39 @@ export interface ComponentToken {
...
+  /**
+   * @desc 指示点之间的间距
+   * @descEN margin of indicator
+   */
+  dotMargin: number;
+  /**
+   * @desc 指示点距离边缘的距离
+   * @descEN dot offset to Carousel edge
+   */
+  dotOffset: number;
```
Comment (MadCcc) [inline components/carousel/style/index.ts:27]: 这个 offset 怎么理解？指示器应该是居中的吧

Comment (afc163) [reply components/carousel/style/index.ts:27]: dots 离 Carousel 边缘的距离。

**Review by kiner-tang — APPROVED** [review]

**Review by zombieJ — APPROVED** [review]

**Pull request merged**